packages/passport/sdk/src/authManager.test.ts

```diff
@@ -170,13 +169,13 @@ describe('AuthManager', () => {
 
     describe('when the user has registered for imx', () => {
       it('should populate the imx object', async () => {
-        mockSigninPopup.mockResolvedValue({
-          ...mockOidcUser,
-          profile: {
-            ...mockOidcUser.profile,
-            passport: {
-              ...imxProfileData,
-            },
+        mockSigninPopup.mockResolvedValue(mockOidcUser);
+        // eslint-disable-next-line global-require
+        (require('jwt-decode') as jest.Mock).mockReturnValue({
+          passport: {
+            imx_eth_address: mockUserImx.imx.ethAddress,
+            imx_stark_address: mockUserImx.imx.starkAddress,
+            imx_user_admin_address: mockUserImx.imx.userAdminAddress,
           },
```

Comment:
To avoid having the `require('jwt-decode')` & disabling the linter on the next line, can we instead follow the same pattern that's being used for the `UserManager`? E.g.:

```
import jwt_decode from 'jwt-decode';
...
jest.mock('jwt-decode');
...
(jwt_decode as jest.Mock).mockImplementation(() => ({
  passport: {
    imx_eth_address: mockUserImx.imx.ethAddress,
    imx_stark_address: mockUserImx.imx.starkAddress,
    imx_user_admin_address: mockUserImx.imx.userAdminAddress,
  },
}));
```